profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/compute/ProfitBricksComputeServiceAdapter.java

```diff
@@ -276,20 +274,42 @@ public boolean apply(Lan input) {
          }
       }
 
+      List<FirewallRule> firewallrules = new ArrayList<FirewallRule>();
+      boolean firewallActive = false;
+      if (inboundPorts.length > 0) {
+         firewallActive = true;
+      }
+
       Nic nic = api.nicApi().create(Nic.Request.creatingBuilder()
               .dataCenterId(dataCenterId)
               .name("jclouds" + name)
               .dhcp(Boolean.TRUE)
               .lan(lanId)
-              .firewallActive(Boolean.FALSE)
+              .firewallActive(firewallActive)
```

Comment:
Just `.firewallActive(inboundPorts.length > 0)` and remove the previous conditional.